### ISSUE TITLE
Flip Cards on Hover

### ISSUE THREAD

**Issue by buddy-harlow** [Bixal/bixal-site-drupal#240]
### Repeated/duplicate headings
- Every card heading (h3) is listed twice, see the ANDI screenshot below.
### The revealed content (from flip) is not keyboard accessible.
- Hover functionality is only available with mouse.
- Fails the dismissible criteria of WCAG SC 1.4.13 (AA)
## Suggestion
Removing/disabling the flip transition and only having the back of the cards be present.

**Comment by mejiaj** [comment]
Moving this back to `Todo` because we're going to add capabilities **without** using the flip cards.

**Comment by mejiaj** [comment]
This will be closed by #386 and #387.
